mobile-app/app/screens/AppNavigator/screens/Auctions/screens/PlaceBidScreen.tsx

```diff
@@ -135,130 +135,230 @@ export function PlaceBidScreen(props: Props): JSX.Element {
+          </View>
+          <ThemedViewV2
+            light={tailwind("bg-mono-light-v2-00")}
+            dark={tailwind("bg-mono-dark-v2-00")}
+            style={tailwind(
+              "flex flex-row justify-around items-center py-3 mt-6 mx-1 rounded-xl-v2 font-normal-v2"
+            )}
+          >
+            {Object.values(BidAmountButtonTypes).map(
+              (percentageAmount, index, { length }) => {
+                return (
+                  <BidAmountButton
+                    key={percentageAmount}
+                    onPress={onBidPercentagePress}
+                    bidPercentageAmount={percentageAmount}
+                    minNextBidInToken={minNextBidInToken.toFixed(8)}
+                    hasBorder={length - 1 !== index}
+                  />
+                );
+              }
+            )}
+          </ThemedViewV2>
 
-          {displayHigherBidWarning && (
-            <InfoText
-              testID="conversion_info_text"
-              text={translate(
+          {(displayMinBidError || displayMinBidMessage) && (
+            <ThemedTextV2
+              light={tailwind("text-mono-light-v2-500", {
+                "text-red-v2": displayMinBidError,
+              })}
+              dark={tailwind("text-mono-dark-v2-500", {
+                "text-red-v2": displayMinBidError,
+              })}
+              style={tailwind("text-xs pt-2 mx-6 font-normal-v2")}
+              testID="min_next_bid_text"
+            >
+              {translate(
                 "screens/PlaceBidScreen",
-                "The value of the tokens you are placing is considerably higher than the total auction value."
+                "The minimum next bid is {{amount}} {{symbol}} (100%)",
+                {
+                  amount: minNextBidInToken.toFixed(8),
+                  symbol: batch.loan.symbol,
+                }
               )}
-              style={tailwind("mt-5")}
-            />
+            </ThemedTextV2>
+          )}
+          {formState.errors.bidAmount?.type === "hasSufficientFunds" && (
+            <ThemedTextV2
+              light={tailwind("text-red-v2")}
+              dark={tailwind("text-red-v2")}
+              style={tailwind("text-xs pt-2 mx-6 font-normal-v2")}
+              testID="insufficient_balance_text"
+            >
+              {translate("screens/PlaceBidScreen", "Insufficient balance")}
+            </ThemedTextV2>
+          )}
+          {displayHigherBidWarning && formState.isValid && (
+            <ThemedTextV2
+              light={tailwind("text-orange-v2")}
+              dark={tailwind("text-orange-v2")}
+              style={tailwind("text-xs pt-2 mx-6 font-normal-v2")}
+              testID="high_bid_text"
+            >
+              {translate(
+                "screens/PlaceBidScreen",
+                "Your bid is higher than the auction's collateral value of {{prefix}}{{amount}}",
+                { prefix: "$", amount: totalCollateralsValueInUSD.toFixed(8) }
```

Comment:
should use getPrecisedTokenValue() when display `$` value for token

Reply:
Updated.